modules/src/main/java/eu/itesla_project/modules/contingencies/ContingenciesAndActionsDatabaseClientFactory.java

```diff
@@ -25,4 +26,8 @@ default ContingenciesAndActionsDatabaseClient create(Path contingenciesAndAction
         return create();
     }
 
+    @Override
+    default ContingenciesAndActionsDatabaseClient create(InputStream data) {
```

Comment:
so we should also remove the default implementation of  create(Path contingenciesAndActionsFile)  method ?